Makefile

```diff
@@ -35,8 +35,10 @@ TESTFLAGS    :=
 STRESSFLAGS  :=
 DUPLFLAGS    := -t 100
 BUILDMODE    := install
+GOPATH       := $(realpath $(dir $(realpath $(lastword $(MAKEFILE_LIST))))/../../../..)
```

Comment:
`make` always runs from the directory of the `Makefile`, so you should be able to do `$(realpath ../../../..)` directly (we do this in `ui/Makefile`).